opensrp-chw/src/ba/assets/json.form/pnc_health_facility_visit.json

```diff
@@ -155,6 +155,13 @@
         "read_only": "true",
         "hidden": "true",
         "value": "0"
+      },
+      {
+        "key": "pnc_hf_next_visit_date",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "anc_hf_next_visit_date",
```

Comment:
Should you be using **anc_hf_next_visit_date** as the entity ID? This should be the **pnc_hf_next_visit_date**

Reply:
I'll remove the anc_hf_next_visit_date. Wrongly copied this over from the anc_hf_next_visit_date